internal/application/constants/dbconstants.go

```diff
@@ -26,4 +26,26 @@ var (
 		Query: "SELECT CONSUMER_KEY, CONSUMER_SECRET, CALLBACK_URIS, GRANT_TYPES " +
 			"FROM IDN_OAUTH_CONSUMER_APPS WHERE CONSUMER_KEY = $1",
 	}
+	QueryGetApplicationByAppId = dbmodel.DBQuery{
+		Id: "ASQ-00002",
```

Comment:
Can we introduce DB type specific queries too ?

Reply:
We should be able to do that. We will add that support when we have a concrete requirement at hand.